utilities/docker/geppetto-persistence-frontend/geppetto.plan

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plan name="multi-artifact.plan" version="1.0.0" scoped="true" atomic="true"
+        xmlns="http://www.springsource.org/schema/dm-server/plan"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="
+            http://www.springsource.org/schema/dm-server/plan
+            http://www.springsource.org/schema/dm-server/plan/springsource-dm-server-plan.xsd">
+
+
+    <artifact type="bundle" name="org.geppetto.model" version="0.3.3"/>
```

Comment:
@tarelli I don't think so. It's only used inside the dockerfile in one line "COPY geppetto.plan workspace/org.geppetto/", but I think it gets overwritten once org.geppetto contents are checkout from github